pkgs/by-name/rt/rtorrent/package.nix

```diff
@@ -14,17 +14,18 @@
 , xmlrpc_c
 , zlib
 , nixosTests
+, unstableGitUpdater
 }:
 
 stdenv.mkDerivation {
```

Comment:
Split outputs:

```nix
   outputs = [ "out" "man" ];
```

Reply:
Done, also using `installManpage` instead of manually installing manpages and using a newer `rtorrent.rc-example` instead of the old one.